discovery-provider/src/tasks/entity_manager/notification.py

```diff
@@ -66,31 +66,12 @@ def view_notification(params: ManageEntityParameters):
     params.add_notification_seen_record(key, notification_seen)
 
 
-def get_notification_user_ids(
```

Comment:
couldn't find another place where this was used, since it's only for announcements i figured we'd remove the temptation of using it entirely